README.md

```diff
@@ -277,6 +277,38 @@ sys.addShutdownHook(system.terminate())
 ```
 
 ## Changelog  
+
+### 3.0.1 (2017-10-27)
+Changes since v3.0.0
+  - Akka 2.5 release only, please use the 2.7.x release for Akka 2.4 
+  - Changed oracle test-docker image
```

Comment:
In my opinion, changes to the tests should do not need to be included in the release notes.
The same holds for "Cache dependencies in Travis CI."

Reply:
Sure. I will remove it